operator/v1/types_ingress.go

```diff
@@ -1388,6 +1388,27 @@ type IngressControllerTuningOptions struct {
 	// +kubebuilder:validation:Format=duration
 	// +optional
 	HealthCheckInterval *metav1.Duration `json:"healthCheckInterval,omitempty"`
+
+	// maxConnections defines the maximum number of simultaneous
+	// connections that can be established per HAProxy process.
+	// Increasing this value allows each ingress controller pod to
+	// handle more connections but at the cost of additional
+	// system resources being consumed.
+	//
+	// If this field is empty or 0, the IngressController will use
+	// the default value of 20000, but the default is subject to
+	// change in future releases. If the value is -1 then HAProxy
+	// will dynamically compute a value based on ulimits and the
+	// the configuration file when the haproxy process starts.
```

Comment:
What configuration file does this refer to?

Reply:
haproxy.config; followed up with a change that mentions this configuration file by name.